### DISCUSSION TITLE
my github copilot on VS code is not working  it says im not authorized

### DISCUSSION THREAD

**Discussion by omaerxd** [community/community#14778] [Copilot Conversations] [answered]
my github copilot on VS code is not working   here's my problem:

[INFO] [auth] [2022-04-12T11:29:56.317Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-04-12T11:29:56.321Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

**Comment by kafka-yu** [comment, marked as answer]
+1

**Reply by davecheney** [reply]
@kafka-yu I can see that you are still on the waitlist. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
